packages/gatsby-theme-carbon/src/components/AccordionItem/AccordionItem.js

```diff
@@ -0,0 +1,152 @@
+/**
```

Comment:
Definitely don't want to be copy and pasting carbon-components-react source code. This will break if we make any changes to the react or carbon components libraries.

Try, instead, targeting the classNames exposed by default from the react library from your own style sheets. You can supply a custom class name to the accordion item, then target the title, content, heading etc from the style sheet.

```
.accordionItem :global(bx--accordion__title) {
	// title styles go here
}
```

Reply:
Ah, thank you for this1 This was at the root of the question I was trying to ask the other day - how to target the internal classes without explicitly adding them into the react component itself.

Reply:
Supply custom class name then target elements as needed.